Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -515,6 +515,8 @@ Parser::ParseErrorOr<NonnullRefPtr<StyleValue const>> Parser::parse_css_value(Pr
         //       values, only the CSS-wide keywords - this is handled above, and thus, if we have gotten to here, there
         //       is an invalid value which is a syntax error.
         return ParseError::SyntaxError;
+    case PropertyID::AccentColor:
+        return parse_all_as(tokens, [this](auto& tokens) { return parse_accent_color(tokens); });
```

Comment:
The parsing changes also aren't needed, it's simple enough for the parser to do automatically. (For what it's worth, your parsing code is good, it just isn't needed.)

Reply:
Done